argo/providers/vai/eventing_server.go

```diff
@@ -77,7 +77,7 @@ func (es *VaiEventingServer) StartEventSource(source *generic.EventSource, strea
 			return
 		}
 
-		es.Logger.V(1).Info("sending run completion data", "event", event)
+		es.Logger.Info("sending run completion data", "event", event)
```

Comment:
The `V(1)` sets the verbosity to a lower priority than a normal Info log, so it wasn't showing up in the logs. I'm not sure if it had a purpose, but removing it makes the log show up, so 🤷